src/cpp/main.cpp

```diff
@@ -47,32 +51,48 @@ size_t initialize(const Config *config) {
   displacements = (Vector*) calloc(num_particles, sizeof(Vector));
 
   // Initialize the particles.
+
+  // Particles radious generator in milimeters
+  int seed = 7;
+  std::mt19937 generator(seed);
+  std::uniform_real_distribution<double> uniform_distribution(min_radious, max_radious);
+
+  // In README.md says that m is the parameter for the Y axis ??
```

Comment:
Oh really, my fault, sorry. Could you please fix that?

Reply:
So, you mean that what is wrong is the README, not the code?

Reply:
Yes, the README is wrong. `n` is for `y` and `m` for `x`.